### PR TITLE
Add glog 0.7.0

### PR DESCRIPTION
**Pull request by Vertexwahn**
Add glog 0.7.0

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (glog) have been updated in this PR. Please review the changes.

**Comment by Vertexwahn** [comment]
FYI: There is no stable source URL

File: modules/glog/0.7.0/source.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "integrity": "sha256-N1EGtZdiMbkuZoecGpLOBikjua5XPEK1a6KLES7kzBE=",
+    "patch_strip": 1,
```
Comment (meteorcloudy) [inline modules/glog/0.7.0/source.json:3]: This should be 0

```

can't find file to patch at input line 3
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|--- MODULE.bazel
|+++ MODULE.bazel
--------------------------
File to patch:
```

**Review by meteorcloudy — CHANGES_REQUESTED** [review]

File: modules/glog/metadata.json [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
-  ],
-  "versions": [
-    "0.5.0",
-    "0.6.0"
-  ],
-  "yanked_versions": {}
+    "homepage": "https://github.com/google/glog",
+    "maintainers": [
+        {
+            "email": "bcr-maintainers@bazel.build",
```
Comment (meteorcloudy) [inline modules/glog/metadata.json:5]: Do you mind adding yourself as maintainer here?

**Review by meteorcloudy — APPROVED** [review]

File: modules/glog/metadata.json (hunk 2 above)
Comment (Vertexwahn) [reply modules/glog/metadata.json]: Added me as a maintainer

**Comment by Vertexwahn** [comment]
@meteorcloudy I think this is ready to go!

**Pull request merged**